### ISSUE TITLE
Wayland support

### ISSUE THREAD

**Issue by Tarik02** [MCMrARM/mcpelauncher-linux#383]

**Comment by MCMrARM** [comment]
I believe GLFW should have Wayland support, try to use it instead of EGLUT?

**Comment by Tarik02** [comment]
Okay, but i can't compile it with glfw and cef together.

**Comment by Tarik02** [comment]
I changed some places and compiled. But there is no wheel button handling, when i scroll the hotbar item isn't changed.

**Comment by KennFatt** [comment]
Everything its fine with Wayland. And I use EGLUT for it.

**Comment by Tarik02** [comment]
@KennFatt, so it uses Xwayland and it is legacy way. There is needed a modern way like glfw with wayland support.

**Comment by KennFatt** [comment]
Does it really matter?

**Comment by Tarik02** [comment]
@KennFatt yes, Xwayland is going to be optional (default removed) in few decades. Because X is legacy already.

**Comment by MCMrARM** [comment]
I'll likely try to implement wayland support in eglut after I switch to wayland myself finally, but it's not a priority.

As for Xbox Live login with GLFW, you'll need to wait until the project refactor/restructurization is completed.

**Comment by Enverex** [comment]
>Because X is legacy already.

This is not true in the slightest. A few people trying to push Wayland over XOrg does not make the latter "legacy".

**Comment by tarik02** [comment]
I mean it is legacy as for me.

**Comment by MCMrARM** [comment]
It's already possible to get Xbox Live with the rewritten launcher, and I'm willing to help with getting Wayland under GLFW here, join the discord if you're interested; https://discord.gg/TaUNBXr
